### ISSUE TITLE
resource yandex_datatransfer_transfer doesn't support import

### ISSUE THREAD

**Issue by bandikoot** [yandex-cloud/terraform-provider-yandex#230]
```
$ terraform import yandex_datatransfer_transfer.name <existent_transfer_id>
yandex_datatransfer_transfer.name: Importing from ID "<existent_transfer_id>"...

Error: resource yandex_datatransfer_transfer doesn't support import
```

However, [it is stated in the provider documentation](https://registry.terraform.io/providers/yandex-cloud/yandex/latest/docs/resources/datatransfer_transfer#import) that import of `yandex_datatransfer_transfer` resource is possible. It would be nice to fix provider's behavior according to docs.

Importing `yandex_datatransfer_endpoint` resources is ok, btw.

**Comment by bandikoot** [comment]
@ovandriyanov, using the apportunity — thanks for adding these resources in the first place (:

**Comment by bandikoot** [comment]
Seems to be fixed in [v0.71.0](https://github.com/yandex-cloud/terraform-provider-yandex/releases/tag/v0.71.0)

Found the commit with the fixes — [a2dc2e8](https://github.com/yandex-cloud/terraform-provider-yandex/commit/a2dc2e8510638498e19085864027133e6c067f05)
